examples/one-2-one/.gitignore

```diff
@@ -0,0 +1,64 @@
+# Logs
```

Comment:
We don't keep package-level `.gitignore` files and use a top-level `.gitignore` file in loopback-next monorepo instead. Please remove this file.

Reply:
ok